.cursor/commands/commit-push-pr.md

```
@@ -0,0 +1,221 @@
+git add -A && \
+git commit -m "$MSG" && \
+git push -u origin "$BRANCH"
+
+# ここでAIがPR作成を実行（例）
+# - ブランチ名から目的を推測
+# - git diff --name-status で変更ファイルを確認
+# - PRタイトルとメッセージを自動生成
+# - mcp_github_create_pull_request / gh pr create 等で PR 作成
+```
+
+> 注意: MCP（Model Context Protocol）はエージェントからGitHub等の外部サービスを安全に操作するための標準プロトコルです。本手順ではPR作成にMCPのGitHub連携を使用します。利用にはMCP対応環境の設定が必要です。参考: https://modelcontextprotocol.io/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)